lib/ontologies_linked_data/models/mod/semantic_artefact_catalog.rb

```diff
@@ -0,0 +1,228 @@
+require 'yaml'
+
+module LinkedData
+    module Models
+        class SemanticArtefactCatalog < LinkedData::Models::Base
+
+
+            # Load the scheme file once
+            SCHEME_PATH = File.join(__dir__, '../../../../config/schemes/semantic_artefact_catalog.yml')
+            SCHEME = YAML.load_file(SCHEME_PATH)
+
+            class << self
+                def define_attribute(name, **options)
```

Comment:
no need just use regular attribute

Reply:
i did i want to define the default value in the yaml file, and this function will make the default values to be a lambda functions or proc